cosmoshub/assetlist.json

```diff
@@ -866,6 +866,29 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/paxg.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/paxg.svg"
       }
+    },
+    {
+    "description": "Quark ($QUARK) is the first permissionless CosmWasm token launched on the Cosmos Hub.",
+    "denom_units": [
+      {
+        "denom": "factory/cosmos1n6asrjy9754q8y9jsxqf557zmsv3s3xa5m9eg5/quark",
```

Comment:
This contract is incorrect (should be factyory/osmo..., not factory/cosmos...) and relocated to the Osmosis assetlist. The description is also misleading for tyhe same reason.